backend/usermanagement/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from django.db import models
+from django.contrib.auth.models import User
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    profile_picture = models.ImageField(upload_to='profile_pics', default='default.jpg')
+
+    def __str__(self):
+        return f'{self.user.username} Profile'
+    
+class Games(models.Model):
```

Comment:
tbd how this fits with tournament and matchmaking apps from an architecture point of view